### PR TITLE
Remove unnecessary import

### PR DESCRIPTION
**Pull request by srawlins**
The analyzer will start reporting this when dart-lang/sdk#44569 is implemented.

### CODE REVIEW

File: lib/src/json_document_transformer.dart [hunk 1]
```diff
@@ -7,7 +7,6 @@ import 'dart:convert';
 import 'package:async/async.dart';
 
 import '../stream_channel.dart';
```
Comment (kevmoo) [inline lib/src/json_document_transformer.dart:9]: Can you switch these? Better to not import the 'root' library...

Comment (srawlins) [reply lib/src/json_document_transformer.dart:9]: Unfortunately / oddly, `class StreamChannel` is declared in `lib/stream_channel.dart`, and `lib/stream_channel.dart` exports the lib/src stuff. So we _must_ import the root library.

Comment (kevmoo) [reply lib/src/json_document_transformer.dart:9]: Sad... 😞

**Review by kevmoo — APPROVED** [review]

File: lib/src/json_document_transformer.dart (hunk 1 above)
Comment (srawlins) [reply lib/src/json_document_transformer.dart:9]: It is... odd. I think a non-breaking change to move the declaration to src/ and export from lib/stream_channel.dart.

**Pull request merged**